.github/workflows/prepare-release.yml

```
@@ -0,0 +1,86 @@
+name: Prepare Shared GitHub Actions for Release
+
+on:
+  push:
```

Does the `push` event need to be removed here?

```suggestion
  push:
```